reframe/frontend/executors/policies.py

```diff
@@ -204,14 +228,16 @@ def on_task_run(self, task):
         self._running_tasks.append(task)
 
     def on_task_failure(self, task):
+        mssg = f'{task.check.info()} {format_duration(task.duration)}'
         if task.failed_stage == 'cleanup':
-            self.printer.status('ERROR', task.check.info(), just='right')
+            self.printer.status('ERROR', mssg, just='right')
         else:
             self._remove_from_running(task)
-            self.printer.status('FAIL', task.check.info(), just='right')
+            self.printer.status('FAIL', mssg, just='right')
 
     def on_task_success(self, task):
-        self.printer.status('OK', task.check.info(), just='right')
+        mssg = f'{task.check.info()} {format_duration(task.duration)}'
+        self.printer.status('OK', mssg, just='right')
```

Comment:
You will need to  print another status line here with all the timings at verbose level.